src/styles/Books.css

```diff
@@ -0,0 +1,239 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:wght@400;600;800;900&display=swap');
+

```

Comment:
- [ ] I recommend sticking the Zeplin design as much as possible. For example, I can see the differences in spacing, colors, font size, etc. Also, I marked them with yellow color in the below screenshot 👇 
- Original 👇 
![Screenshot (202)](https://user-images.githubusercontent.com/67087664/155662566-fcbf13c0-2543-42bc-a602-8a23c0fba84b.png)

- Yours 👇 
![Screenshot (204)_LI](https://user-images.githubusercontent.com/67087664/155662445-ba470eec-b67a-483d-8797-fb50e157cebc.jpg)
1. you forgot to put `bar`s between sections;
2. navbar titles should not have `underlines`; 
3. navbar `icon` which is on the right side looks different from the given design;
4. `ADD BOOK` text color should be white;
5. `Update progress` text should be as in the given design;
6. other small details should match the design;